### ISSUE TITLE
3.0.0: links via stand-alone @see tags are broken

### ISSUE THREAD

**Issue by jrfnl** [phpDocumentor/phpDocumentor#2422]
_Loosely related to closed issue #1726._

**Given the following piece of code in file `src/issue-2422.php`:**
``` php
<?php
/**
 * Class summary
 *
 * A description containing an inline {@see \Issue2422} tag and another {@see \Issue2422::$property}.
 *
 * @see \Issue2422            Class
 * @see \Issue2422::CONSTANT  Constant
 * @see \Issue2422::$property Property
 * @see \Issue2422::method    Method
 */
class Issue2422
{
    const CONSTANT = 'foo';

    public $property;

    public function method() {}
}
```

<details>
  <summary><b>with the following config:</b></summary>

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<phpdocumentor
    configVersion="3"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns="https://www.phpdoc.org"
    xsi:noNamespaceSchemaLocation="https://docs.phpdoc.org/latest/phpdoc.xsd"
>
    <title>Demo</title>
    <version number="latest">
        <api format="php">
            <source dsn=".">
                <path>src</path>
            </source>
        </api>
    </version>
    <template name="default"/>
</phpdocumentor>
```
</details>

## Expected behavior
I'd expect the links generated by the stand-alone `@see` tags to be functional.

## Actual behavior
The links are broken and result in a 404.

HTML output for the broken stand-alone `@see ...` tag (indentation tidied up for readability):
```html
<dt class="phpdocumentor-tag-list__entry">
	<span class="phpdocumentor-tag__name">see</span>
</dt>
<dd class="phpdocumentor-tag-list__definition">
	<span class="phpdocumentor-tag-link"><a href="../\Issue2422"><abbr title="\Issue2422">Issue2422</abbr></a></span>
	<section class="phpdocumentor-description"><p>Class</p></section>
</dd>
<dt class="phpdocumentor-tag-list__entry">
	<span class="phpdocumentor-tag__name">see</span>
</dt>
<dd class="phpdocumentor-tag-list__definition">
	<span class="phpdocumentor-tag-link"><a href="\Issue2422::CONSTANT"><abbr title="\Issue2422::CONSTANT">Issue2422::CONSTANT</abbr></a></span>
	<section class="phpdocumentor-description"><p>Constant</p></section>
</dd>
<dt class="phpdocumentor-tag-list__entry">
	<span class="phpdocumentor-tag__name">see</span>
</dt>
<dd class="phpdocumentor-tag-list__definition">
	<span class="phpdocumentor-tag-link"><a href="\Issue2422::$property"><abbr title="\Issue2422::$property">Issue2422::$property</abbr></a></span>
	<section class="phpdocumentor-description"><p>Property</p></section>
</dd>
<dt class="phpdocumentor-tag-list__entry">
	<span class="phpdocumentor-tag__name">see</span>
</dt>
<dd class="phpdocumentor-tag-list__definition">
	<span class="phpdocumentor-tag-link"><a href="\Issue2422::method"><abbr title="\Issue2422::method">Issue2422::method</abbr></a></span>
	<section class="phpdocumentor-description"><p>Method</p></section>
</dd>
```

Note: the links generated via the inline `{@see ..}` tags in the long description **_do_** work correctly.

HTML output for the working inline `{@see ...}` tag:
```html
<section class="phpdocumentor-description"><p>A description containing an inline <a href="../classes/Issue2422.html">\Issue2422</a> tag and another <a href="../classes/Issue2422.html#property_property">\Issue2422::$property</a>.</p></section>
```

## As found in project

* [Direct link to the source code where the issue was found](https://github.com/PHPCSStandards/PHPCSUtils/blob/develop/PHPCSUtils/Tokens/Collections.php) (tags are used throughout the file)
* [Direct link to the HTML source of the generated docs](https://github.com/PHPCSStandards/PHPCSUtils/blob/develop/docs/phpdoc/classes/PHPCSUtils-Tokens-Collections.html)
* [Output as seen online](https://phpcsutils.com/phpdoc/classes/PHPCSUtils-Tokens-Collections.html)
    _Note: the online docs have not been updated recently, but will be soon._

## Steps to reproduce the problem

1. Issue can be reproduced with the above code sample and config or via the linked project.
2. Run `php -f /path/to/phpDocumentor/bin/phpdoc`

## Your environment

* Version used: phpDocumentor v3.0.x-dev@d15f2d6
* Install method: git clone
* PHP version: 7.4.6
* Operating system and version: Windows 7
* Link to your project: https://github.com/PHPCSStandards/PHPCSUtils/
* Link to output as committed: https://github.com/PHPCSStandards/PHPCSUtils/tree/develop/docs/phpdoc
* Link to output online: https://phpcsutils.com/phpdoc/

**Comment by mvriel** [comment]
Found the cause for this issue in the LinkRenderer. The renderer did not take the Reference object into account. References, as used by `@see`, are allowed to be either a URL or FQSEN. But by default, the LinkRenderer assumes that a passed value is a URL.

In commit 07b1dc7891f34401da5167ea0f81af3092f4abca, I have added an extra clause to take the Reference/FQSEN element into account and generate a correct URL with it
